lib/matplotlib/artist.py

```diff
@@ -534,9 +534,14 @@ def pick(self, mouseevent):
                           mouseevent, self, **prop)._process()
 
         # Pick children
-        for a in self.get_children():
+        children = self.get_children()
+        for a in children:
             # make sure the event happened in the same Axes
             ax = getattr(a, 'axes', None)
+            # if subfigure must get the axes from it
+            if isinstance(a, mpl.figure.SubFigure):
+                children.extend(a.get_children())
```

Comment:
I'm not 100% sure this is safe, and it may not have the effect you'd want everywhere.